torch_lr_finder/lr_finder.py

```diff
@@ -6,6 +6,16 @@
 from torch.optim.lr_scheduler import _LRScheduler
 import matplotlib.pyplot as plt
 
+try:
+    from apex import amp
+    IS_AMP_AVAILABLE = True
+except ImportError:
+    import logging
+    logger = logging.getLogger(__name__)
```

Comment:
Is there a specific use-case for using `logging` here?

Also, `logging.getLogger(__name__)` fails on Python 2 and outputs: `No handlers could be found for logger "torch_lr_finder.lr_finder"`

Reply:
In my opinion, using a logger to show that message makes users able to silence it once they want to do so. And user can configure the logger before `import torch_lr_finder`, like the code as follows:
```python
# set logging level to ERROR to silence the warning message
import logging
logging.getLogger("torch_lr_finder.lr_finder").setLevel(logging.ERROR)

# warning message "To enable mixed precision training..." won't be shown if apex is not installed
from torch_lr_finder import LRFinder
```

As for the message `No handlers could be found for logger "torch_lr_finder.lr_finder"`, it seems like we have to manually call `logging.basicConfig()` before `logging.getLogger()` in Python 2. I'll add it into the code.